package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {
     "algoliasearch-helper": "^3.1.0",
-    "instantsearch.js": "^4.16.1"
+    "instantsearch.js": "https://pkg.csb.dev/algolia/instantsearch.js/commit/59ed228e/instantsearch.js"
```

Comment:
this is temporary. let's not merge this until we release it in InstantSearch.js.